### ISSUE TITLE
SDL_Sound for compressed imgdisks disabled?

### ISSUE THREAD

**Issue by NRosenbergLoyaltyOne** [libretro/dosbox-libretro#123]
I'm trying to run a game that has a compressed bin/cue with an ogg track. Both this and the SVN version give me an error. I wonder if SDL_Sound  is not enabled and, if so, would it be possible?

Thank you

**Comment by randombyt** [comment]
I am also facing problems in mounting cd images with compressed audio. The error message is "MSCDEX: Invalid file or unable to open". The images in question do work in stand-alone versions of dosbox (modified).

A brief search of dosbox resources brings out following two leads:
1- https://www.dosbox.com/wiki/Cuesheet (last part discusses compressed audio)
2- https://www.dosbox.com/wiki/Building_DOSBox_with_MinGW (under the heading: Adding support for compressed audio on diskimages - Optional)

Please add support for compressed audio on disk images in dosbox. I requested the same on dosbox-svn repo, sorry for repetition but I am not sure if these cores will merge or not.

**Comment by niclashoyer** [comment]
I am also facing this issue. I try to start [Ignition](https://www.dosbox.com/comp_list.php?showID=2114&letter=I) which needs at least an empty audio disc mounted. But using `IMGMOUNT` fails, even with an empty dummy cue file.

**Comment by niclashoyer** [comment]
After digging into the code, there seems to be no easy way to integrate `SDL_sound` into the libretro port, because there is no real SDL integration, just a [fake wrapper](https://github.com/libretro/dosbox-libretro/blob/master/include/SDL.h) to provide minimal compatibility.

**Comment by NRosenbergLoyaltyOne** [comment]
That is unfortunate, I guess it was never implemented

**Comment by niclashoyer** [comment]
Currently looking into this [other port dosbox-svn](https://github.com/libretro/dosbox-svn), which uses SDL by default (the fake implementation is optional, used by some architectures). It is just using SDL and SDL_net by default, but it seems more likely to include SDL_sound there.

**Comment by niclashoyer** [comment]
I propose to close this here, because it seems unlikely, and continue discussion on the feature here: https://github.com/libretro/dosbox-svn/issues/38

**Comment by NRosenbergLoyaltyOne** [comment]
Agree, it looks like there is also a new svn version core `dosbox-svn-ce`, but I could not find the repo on the libretro github.

**Comment by 2m** [comment]
I have also encountered this problem. Many DOS games from GoG come with compressed audio tracks. So I had to reverse the compression to workaround this.

First, I used `cdemu` to mount the `ins` (same as `cue`) files to an emulated CD drive. I had to move the `ogg` files from the subdirectory to the same directory as `ins` and `gog` (same as `bin`) files.

Once the image was mounted, I used the `cdrdao` command from the [DOSBox wiki](https://www.dosbox.com/wiki/Cuesheet) to read the contents of the virtual CD drive back to an image.

    cdrdao read-cd --datafile image.bin --driver generic-mmc:0x20000 --device /dev/cdrom --read-raw image.toc

This image was successfully mounted by both `dosbox-libretro` and ` dosbox-svn` cores and music was available in games.
